src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/HmppsAccreditedProgrammesApi.kt

```diff
@@ -1,11 +0,0 @@
-package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi
-
-import org.springframework.boot.autoconfigure.SpringBootApplication
-import org.springframework.boot.runApplication
-
-@SpringBootApplication()
-class HmppsAccreditedProgrammesApi
-
-fun main(args: Array<String>) {
```

Comment:
Ah, so this is why it was so hard to find the `main` method in the example project we looked at?

Reply:
Yes, it is generated by the plugin and if you add another main method then you get an error